src/routes/landing/components/CityEntry.js

```diff
@@ -49,21 +51,23 @@ class CityEntry extends React.PureComponent<PropsType> {
     const { city, language, filterText, theme } = this.props
     const normalizedFilter = normalizeSearchString(filterText)
     const aliases = this.getMatchedAliases(city, normalizedFilter)
-
     return (
       <CityListItem to={new CategoriesRouteConfig().getRoutePath({ city: city.code, language })}>
         <Highlighter searchWords={[filterText]} sanitize={normalizeSearchString} aria-label={city.name}
-                     textToHighlight={city.name} highlightStyle={{ backgroundColor: theme.colors.themeColor }} />
+                     textToHighlight={city.name}
+                     highlightStyle={{ backgroundColor: theme.colors.backgroundColor, fontWeight: 'bold' }} />
```

Comment:
```suggestion
                     highlightStyle={{ fontWeight: 'bold' }} />
```

No need to keep the background color here.